src/pages/api/health.ts

```diff
@@ -0,0 +1,54 @@
+import { gql } from "urql";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { createServerClient } from "@/lib/create-graphq-client";
+import { logger } from "@/lib/logger";
+import { env } from "@/lib/env.mjs";
+
+enum HealthEnum {
+  DOWN = "DOWN",
+  UP = "UP",
+}
+
+enum HealthCodeEnum {
+  UP = 200,
+  DOWN = 503,
+}
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const query = gql`
+    query test {
+      __typename
+    }
+  `;
+
+  try {
+    const token = "";
+    const client = createServerClient(env.SALEOR_API_URL as string, token);
```

Comment:
Whats the purpose of adding Apollo Server to this repository? If I'm understanding it correctly, so far its only used to create gql client, and for that you can use urql which is already part of the repo (src/lib/create-graphq-client.ts)